benchmark/microbenchmark/src/androidTest/java/com/apollographql/apollo3/benchmark/ApolloStoreIncubatingTests.kt

```diff
@@ -0,0 +1,98 @@
+package com.apollographql.apollo3.benchmark
+
+import androidx.benchmark.junit4.BenchmarkRule
+import androidx.benchmark.junit4.measureRepeated
+import androidx.test.platform.app.InstrumentationRegistry
+import com.apollographql.apollo3.api.json.jsonReader
+import com.apollographql.apollo3.api.parseJsonResponse
+import com.apollographql.apollo3.benchmark.Utils.dbName
+import com.apollographql.apollo3.benchmark.Utils.operationBasedQuery
+import com.apollographql.apollo3.benchmark.Utils.resource
+import com.apollographql.apollo3.benchmark.test.R
+import com.apollographql.apollo3.cache.normalized.incubating.ApolloStore
+import com.apollographql.apollo3.cache.normalized.incubating.api.CacheKeyGenerator
+import com.apollographql.apollo3.cache.normalized.incubating.api.CacheResolver
+import com.apollographql.apollo3.cache.normalized.incubating.api.FieldPolicyCacheResolver
+import com.apollographql.apollo3.cache.normalized.incubating.api.MemoryCacheFactory
+import com.apollographql.apollo3.cache.normalized.incubating.api.NormalizedCacheFactory
+import com.apollographql.apollo3.cache.normalized.incubating.api.TypePolicyCacheKeyGenerator
+import com.apollographql.apollo3.cache.normalized.incubating.sql.SqlNormalizedCacheFactory
+import org.junit.Assert
+import org.junit.Rule
+import org.junit.Test
+import java.lang.reflect.Method
+import java.util.concurrent.Executors
+
+class ApolloStoreIncubatingTests {
+  @get:Rule
+  val benchmarkRule = BenchmarkRule()
+
+  @Test
+  fun concurrentReadWritesMemory() {
+    concurrentReadWrites(MemoryCacheFactory())
+  }
```

Comment:
Are those uploaded to datadog automagically using the [`run-benchmarks` script](https://github.com/apollographql/apollo-kotlin/blob/b5faf3cfe02989620f95730b1e68d5db361599d2/scripts/run-benchmarks.main.kts#L5) ? I would say so but I can't remember.

Reply:
I _think_ so from what I read in the script 😅.

Reply:
I'd say yes (and also yes for DataDog) Doesn't hurt to track it :) I'll have a look at the DD conf.